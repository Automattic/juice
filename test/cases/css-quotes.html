<html><body>
<p>woot</p>
</body></html>
