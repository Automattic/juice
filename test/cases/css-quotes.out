<html><body>
<p style="background: url(/woot);">woot</p>
</body></html>
